### PR TITLE
A few updates from the GDI specification changes.

### PR DESCRIPTION
**Pull request by jkwatson**
Waiting on the spec pr to merge before this is complete.

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by jkwatson** [comment]
I guess we should hold off on this until the spec changes get merged. I'll put in a separate PR to update the sources jar classifier.

File: splunk-otel-android/src/main/java/com/splunk/rum/Config.java
Comment (mateuszrzeszutek) [inline splunk-otel-android/src/main/java/com/splunk/rum/Config.java:66]: Should it be `getRumAuthToken()`?

Comment (jkwatson) [reply splunk-otel-android/src/main/java/com/splunk/rum/Config.java:66]: I think it should actually be `getRumAccessToken()`. ;)

Comment (jkwatson) [reply splunk-otel-android/src/main/java/com/splunk/rum/Config.java]: fixed.

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**
